gce/image/build_image.sh

```diff
@@ -65,11 +65,17 @@ while [ $# -gt 0 ]; do
     esac
 done
 
-get_version_from_rpm () {
+get_version_from_local_rpm () {
+  RPM=$1
+  RELEASE=$(rpm -qi $RPM | awk '/Release/ { print $3 }' )
+  VERSION=$(rpm -qi $RPM | awk '/Version/ { print $3 }' )
+  echo "$VERSION-$RELEASE"
+}
+
+get_version_from_remote_rpm () {
     RPM=$1
-    RELEASE=$(rpm -qi $RPM | awk '/Release/ { print $3 }' )
-    VERSION=$(rpm -qi $RPM | awk '/Version/ { print $3 }' )
-    echo "$VERSION-$RELEASE"
+    VERSION=$(dnf provides $RPM | grep "Provide.*=" | awk '{print $5}')
```

Comment:
can we switch to something move easily available, like `yum`, `dnf` only exists in fedora.
for when we want to debug, or run this manually

Reply:
yes, it will give the same results